src/components/button/button.module.scss

```diff
@@ -1,6 +1,6 @@
 @import '../utils/index.scss';
 .request {
-  background-color: black;
+  background-color: green;
```

Comment:
Do we really need green buttons in our app?

Reply:
of course not, but, I tried using diferent classes for understand how does it work

Reply:
You played with it that's great. But we don't need it here. So I propose cleaning it up.

Reply:
:) done